### PR TITLE
docs: node version requirement

### LINKED ISSUE #37: Add node version requirement to readme file
## Expected Behavior

There should be a minimum node version requirement in the readme file. Some tests will fail with node versions less than 14.

## Current Behavior

There is no minimum node version requirement in the readme file.

### PR DESCRIPTION
**Pull request by mikereze**
closes #37

### CODE REVIEW

**Comment by henok-tesfaye** [comment]
@mikereze please follow our pull request template hereafter. 

Also there are two commits in this PR while it should be on.
Screen Shot 2020-11-05 at 10 31 07 PM

File: README.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
+
+
```
Comment (henok-tesfaye) [inline README.md:2]: unrelated changes

File: README.md [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
 ## Available Scripts
 
 In the project directory, you can run:
 
+## Node Version Requirenmt
+
+Node 10.16.0 or later version
```
Comment (henok-tesfaye) [inline README.md:11]:
```suggestion
Node 14.0.0 or later version
```

**Review by henok-tesfaye — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mikereze) [reply README.md:11]: I checked it with 10.16.0. I chose that because create-react-app is compatible with 10.16.0 or later version.

Comment (henok-tesfaye) [reply README.md]: That's not a node version.

**Review by henok-tesfaye — APPROVED** [review]
Well done!
